services/ui_backend_service/data/cache/client/cache_server.py

```diff
@@ -208,7 +208,7 @@ def kill(self):
 
     def _worker_details(self):
         return {
-            'keys': self.request['keys'],
+            'keys': len(self.request['keys']),
```

Comment:
is this only used for logging purposes, so returning the length instead of the keys is not an issue?

Reply:
to answer my own comment, yes, seems to be the case.

Reply:
This is only for logging/informational purposes. This way we can ensure fairly fixed amount of data transported through the pipe instead of highly variable amount of data. This ended up exceeding the buffer in some cases when there were hundreds of artifacts to fetch.